VideoAPI/VideoApi.Services/CvpAzureStorageService.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Storage;
+using Azure.Storage.Blobs;
+using Azure.Storage.Sas;
+using VideoApi.Common.Configuration;
+using VideoApi.Services.Contracts;
+
+namespace VideoApi.Services
+{
+    public class CvpAzureStorageService : IAzureStorageService
```

Comment:
The implementation looks pretty identical to `VhAzureStorageService`. Would suggest a base or any solution to to share the methods

Reply:
done